cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -94,6 +94,15 @@ func extractExecutionState(
 		capabilityIDs := map[interpreter.AddressPath]interpreter.UInt64Value{}
 
 		migrations = []ledger.Migration{
+			// Contracts must be migrated first
+			migrators.CreateAccountBasedMigration(
+				log,
+				nWorker,
+				[]migrators.AccountBasedMigration{
+					migrators.NewStagedContractsMigration(migrators.GetStagedContracts),
+				},
+			),
```

Comment:
Not sure if this is the proper way to do this: We need to run contract migration for all accounts, before running other migrations.